ui/app/css/itcss/components/newui-sections.scss

```diff
@@ -106,7 +106,7 @@ $wallet-view-bg: $wild-sand;
 
 @media screen and (min-width: 769px) {
   .main-container {
-    margin-top: 35px;
+    margin-top: 6.9vh;
```

Comment:
What is this for?

Reply:
Makes the top of the main container responsive with respect to the height of the header, so that it always appears in relative the same position. The height of the header is also based on `vh`: see line 6 of `ui/app/css/itcss/components/header.scss`

Reply:
I see. Let's see how this looks in final visual pass. I think normally apps dont response to height changes.